### ISSUE TITLE
Package Request: <Lazarus - Pascal IDE> 

### ISSUE THREAD

**Issue by userx-bw** [voidlinux/void-packages#2126]
Lazarus Pascal and FPC to go along with is as it is a dependent  
I won't boar you with a long drawn out please please please. thought it'd be great to jusr xbps ' it into Void. thanks!

**Comment by skrzyp** [comment]
We need `fpc` first… :)

**Comment by userx-bw** [comment]
was that to me or a general statment because I added that within the body of the text :)

**Comment by skrzyp** [comment]
Yes, I see. But I meant that we need to think about FPC first, instead of making IDE requests…

But it looks like I'm not only one who want to see Free Pascal packaged in Void, there are some package which requires it on my private want-list (like `hedgewars` or `mads`)

**Comment by userx-bw** [comment]
OIC  just getting ahead of myself I guess -- I'm use to installing the both of them at the same time -- well one after the other actually -- one can't really install them at the same time  -- ;) ...
